content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+
+However, getting an intuitive understanding of the basics is very much possible and highly encouraged. Linear regression is probably the most fundamental of all topics you may learn in statistics, and it's worth thinking about it carefully. To be clear, you will likely re-visit this topic many times, likely over the course of years, adding and refreshing pieces of information that will eventually develop into a solid understanding. here, we attempt only to provide a brief introduction to the basics. 
+
+## What is Regression?
+
+A gross simplification of the process of linear regression would be that if we have a series of points between two continuous variables, we want to find a straight line that goes through as many of those points as possible. 
+
+What does it mean to do such a thing? Well, let's start with reminding ourselves, mathematically, what a line is. A straight line is defined by the equation $$ y = mx + b$$ where $b$ is the y-intercept, $m$ is the slope of the line, and $x$ and $y$ are data values. This means that given a point on the x-axis, $x$, a slope $m$, and a y-intercept $b$, we can find the value $y$ that denotes the y-axis value that corresponds to the x-axis value $x$. 
+
+When we talk about linear regression, we'll probably want to discuss this equation in a slightly alterred form: $$y = \beta_0 + \beta_1 x_1$$. We'll discuss what these terms mean in a moment but they relate exactly to a y-intercept ($\beta_0$), a slope ($\beta_1$) and an x-value. 
+
+## Ordinary Least Squares (OLS)
+
+We mentioned above we're putting a line through some points as best we can. But how do we define this "best as we can"? Well, the version of "as best we can" that will be addressed here is the method of least squares. If we imagine a scatter of plots like this: 
+
+```{r, echo = FALSE, warning = FALSE}
+x = c(1:100)
+y = rep(NA, 100)
+for(i in 1:100) {
+  y[i] = 1.3 * x[i] + 10 + rnorm(1, mean = 0, sd = 23)
+}
+
+df = data.frame(
+  x = x, y = y
+)
+
+library(tidyverse)
+library(ggthemes)
+ggplot(df) + 
+  geom_point(aes(x = x, y = y), fill = "red", colour = "black", shape = 21, 
+             size = 3) + 
+  ggthemes::theme_base()
+```
+
+It's clear that we can't actually make our line go <em>directly</em> through very many of them, but perhaps we can figure out some way to make the line <em> as close to as many points as possible</em>. We'll do that by imagining we draw a line at random, and calculate the distance between every single point and the line itself. These are called <strong>residuals</strong>. Then we'll square each one, and take the sum of all those squares. 
+
+You might ask, why square it? The short answer to that question is that squaring it makes a large number of mathematical tools available to us. 
```

Comment:
The simple answer is that we care about the magnitude of deviation from the line (not the sign) and the reason why it's squared and not an absolute value is for the mathematical tools